src/Task/File/Write.php

```diff
@@ -121,20 +126,49 @@ public function replace($string, $replacement)
      */
     public function regexReplace($pattern, $replacement)
     {
-        $this->body = preg_replace($pattern, $replacement, $this->body);
+        $this->stack[] = array_merge(['_' . __FUNCTION__], func_get_args());
         return $this;
     }
 
+    protected function _textFromFile($contents, $filename) {
+        return $contents . file_get_contents($filename);
+    }
+
+    protected function _replace($contents, $string, $replacement) {
```

Comment:
protected functions shound not start with `_`. It violates PSR standard: https://github.com/php-fig/fig-standards/blob/master/accepted%2FPSR-2-coding-style-guide.md#43-methods